### ISSUE TITLE
Nice viewing of dataframes?

### ISSUE THREAD

**Issue by tlnagy** [JunoLab/atom-julia-client#153]
It's much nicer working with dataframes in IJulia than in the atom client. Would it be possible to add a dataframe panel like Rstudio has? That would be very useful, especially if it some basic interactions (e.g. sorting) implemented.

**Comment by MikeInnes** [comment]
This would certainly be possible (and a great feature to have), but it would take a solid week's work to get a prototype going and much more to make it polished/usable. So this comes under the very indefinite "when I have time / when someone else volunteers / when someone pays Julia Computing" label.

See also [this prototype](https://github.com/mikeinnes/dside), though.

**Comment by FelipeLema** [comment]
Does https://github.com/queryverse/ElectronDisplay.jl help on this?

**Comment by pfitzseb** [comment]
No, but https://github.com/JuliaComputing/TableView.jl does/will.
